packages/normalizr/README.md

```diff
@@ -25,8 +25,8 @@ Normalizr is a small, but powerful utility for taking JSON with a schema definit
 
 ## Documentation
 
-* [Introduction](./docs/introduction.md)
```

Comment:
does this mean links to this page will now be broken?

Reply:
Introduction has always been at / so any url links won't change. (Url is set by path). As long as there are no references to the file in the code base there won't be any issues